docs/specification.md

```diff
@@ -97,9 +216,7 @@ Parameter (in Michelson): X
 * The previous bullet point implies that each `X` must have a field annotations with the corresponding entrypoint name.
 In the definitions below it may be omitted, but it is still implied.
 
-Pseudocode is semi-formally defined as a list of assignments where the last assignment has `entrypoint_name` on the left and its argument type (that maps to `X`) on the right.
-
-Note: pseudocode is provided only for readability.
+Note: Haskell definitions are provided only for readability.
 If Michelson type contradics what's written in pseudocode, the Michelson defition takes precedence.
```

Comment:
If Michelson type contradicts what's written in Haskell definition, the Michelson definition takes precedence.